HDWalletKit/Core/Crypto/Encryption/EIP155Signer.swift

```diff
@@ -0,0 +1,59 @@
+import CryptoSwift
+
+public struct EIP155Signer {
+    
+    private let chainID: Int = 1
+    
+    public func sign(_ rawTransaction: RawTransaction, privateKey: PrivateKey) throws -> Data {
+        let transactionHash = try hash(rawTransaction: rawTransaction)
+        let signiture = try privateKey.sign(hash: transactionHash)
```

Comment:
@impul signature. Pls check all this file

Reply:
done